x-pack/test/functional/services/observability/alerts/common.ts

```diff
@@ -25,6 +25,12 @@ const ACTION_COLUMN_INDEX = 1;
 
 type WorkflowStatus = 'open' | 'acknowledged' | 'closed';
```

Comment:
@ersin-erdal Workflow status leftover. Can we remove this?

Reply:
No unfortunately. 
I tried to remove them as i said but setWorkflowStatusForRow and  setWorkflowStatusFilter methods which requires that type are used in some other places...

Reply:
@ersin-erdal I had the same issue. WorkflowStatus code is nested in many places.